### PR TITLE
SAR-423 - MVP Navbar fixes

### PR DESCRIPTION
**Pull request by eawest2**
# Related Tickets

- [SAR-423](https://amida.atlassian.net/browse/SAR-423)
- 
# How Things Worked (or Didn't) Before This PR

Navbar was visible when you weren't logged in, had a broken reports button, and every button on the bar sent you to 404.

# How Things Work Now (And How to Test)

The sign out button now works, the navbar conditionally renders reports only if you're not in mvp mode, and the dashboard button will actually take you to the dashboard (though since that's the only page, it takes you nowhere.) instead of 404ing.

# Readiness

1. [ ] This PR passes all automated tests.
2. [ ] This PR has no linting errors.
3. [ ] This PR's changes to configuration files have been documented in all appropriate places (such as but not limited to README.md), if applicable.

### CODE REVIEW

File: src/components/Navbars/DashboardNavbar.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 
+const isMvp = process.env.REACT_APP_MVP_SETTING;
```
Comment (anthem123) [inline src/components/Navbars/DashboardNavbar.js:4]: I've found the `.env` doesn't do booleans. It's silly looking, but you have to do this to get it to work.

`const isMvp = process.env.REACT_APP_MVP_SETTING === 'false';`

File: src/layouts/Dashboard.js [hunk 2]
```diff
@@ -77,3 +78,11 @@ export default function Dashboard() {
     </Box>
   );
 }
+
+D3Container.propTypes = {
```
Comment (anthem123) [inline src/layouts/Dashboard.js:82]: Linter is complaining here. `D3Container` should be `Dashboard`

File: src/components/Navbars/DashboardNavbar.js (hunk 1 above)
Comment (eawest2) [reply src/components/Navbars/DashboardNavbar.js]: Dun did.

File: src/layouts/Dashboard.js (hunk 2 above)
Comment (eawest2) [reply src/layouts/Dashboard.js]: Diddidt

**Review by anthem123 — APPROVED** [review]
Perfection, approved.

**Pull request merged**